### PR TITLE
feat: Add Ray on Vertex AI cluster management sample

### PR DESCRIPTION
**Pull request by yinghsienwu**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Add Ray on Vertex AI cluster management sample 
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/ray_on_vertex/ray_cluster_management.ipynb [hunk 1]
```diff
@@ -0,0 +1,638 @@
+{
```
Comment (gericdong) [inline notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Copyright 2023 Google LLC
</pre><p>Update to 2024</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (gericdong) [inline notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: <p>Vertex &gt; Vertex AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gericdong) [inline notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: <p>how to use Ray on Vertex SDK &gt; how to use <strong>the</strong> Ray on Vertex <strong>AI</strong> SDK</p><p>Ray on Vertex overview &gt; Ray on Vertex <strong>AI</strong> overview</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (gericdong) [inline notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: <ul><li>how to creating cluster, listing clusters, getting a cluster, updating (manually scaling) cluster, and deleting cluster. &gt; how to create a cluster, get a cluster, update (manually scale) a cluster, and delete a cluster</li><li>Ray on Vertex &gt; Ray on Vertex AI</li><li>Confirm if the notebook is using Vertex AI Training?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (gericdong) [inline notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: <p>depens &gt; depending</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (yinghsienwu) [reply notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: done

Comment (yinghsienwu) [reply notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: done

Comment (yinghsienwu) [reply notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: done

Comment (yinghsienwu) [reply notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: Good call! I removed Vertex AI Training and keep Ray on Vertex AI.

Comment (yinghsienwu) [reply notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: done

File: notebooks/official/ray_on_vertex/ray_cluster_management.ipynb [hunk 2]
```diff
@@ -0,0 +1,637 @@
...
```
Comment (gericdong) [inline notebooks/official/ray_on_vertex/ray_cluster_management.ipynb:1]: <p>Vertex &gt; Vertex AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (yinghsienwu) [reply notebooks/official/ray_on_vertex/ray_cluster_management.ipynb]: done

**Pull request merged**